README.md

```diff
@@ -231,4 +231,11 @@ const decimalValue2 = convertValueToDecimal('0.005', 8); // 500000 (0.005 coin w
 })()
 ```
 
+### You want to calculate rate /add pool liquidity for a pair of tokens
+```typescript
+(async () => {
+
```

Comment:
Better to add a comment here about go looking on test file and "an example will be available soon", than leave an empty function call.